streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1351,7 +1351,7 @@ private String decorateTopic(final String topic) {
                                             + "setApplicationId first");
         }
         if (hasNamedTopology()) {
-            return getPrefix() + "-" + topologyName + "-" + topic;
+            return getPrefix() + topologyName + "-" + topic;
```

Comment:
Pretty confident this is true, but just to be 100% sure, we're good with the other `"-"` in this naming scheme right?

Reply:
I don't see why not